### ISSUE TITLE
Deprecation warnings in Rails 5

### ISSUE THREAD

**Issue by anthonator** [MindscapeHQ/raygun4ruby#93]
I started getting a slew of deprecation warnings when running tests after installing `raygun4ruby`.

Ruby 2.3.1
Rails 5.0.0

```
DEPRECATION WARNING: Passing strings or symbols to the middleware builder is deprecated, please change
them to actual class references.  For example:

  "ActionDispatch::DebugExceptions" => ActionDispatch::DebugExceptions

 (called from <top (required)> at /usr/src/app/config/environment.rb:5)
DEPRECATION WARNING: Passing strings or symbols to the middleware builder is deprecated, please change
them to actual class references.  For example:

  "Raygun::Middleware::RackExceptionInterceptor" => Raygun::Middleware::RackExceptionInterceptor

 (called from <top (required)> at /usr/src/app/config/environment.rb:5)
DEPRECATION WARNING: Passing strings or symbols to the middleware builder is deprecated, please change
them to actual class references.  For example:

  "Raygun::Middleware::RailsInsertAffectedUser" => Raygun::Middleware::RailsInsertAffectedUser

 (called from <top (required)> at /usr/src/app/config/environment.rb:5)
DEPRECATION WARNING: Passing strings or symbols to the middleware builder is deprecated, please change
them to actual class references.  For example:

  "ActionDispatch::DebugExceptions" => ActionDispatch::DebugExceptions

 (called from <top (required)> at /usr/src/app/config/environment.rb:5)
DEPRECATION WARNING: Passing strings or symbols to the middleware builder is deprecated, please change
them to actual class references.  For example:

  "Raygun::Middleware::RackExceptionInterceptor" => Raygun::Middleware::RackExceptionInterceptor

 (called from <top (required)> at /usr/src/app/config/environment.rb:5)
DEPRECATION WARNING: Passing strings or symbols to the middleware builder is deprecated, please change
them to actual class references.  For example:

  "Raygun::Middleware::RailsInsertAffectedUser" => Raygun::Middleware::RailsInsertAffectedUser

 (called from <top (required)> at /usr/src/app/config/environment.rb:5)
```

**Comment by ezekg** [comment]
Any update on where this is at?

**Comment by UberMouse** [comment]
Looks this issue was fixed in commit c8f98c0165d192586e343f2da83e59422d3e2137 so I'm going to close this
